src/main/java/org/spongepowered/api/data/manipulator/mutable/tileentity/FurnaceData.java

```diff
@@ -36,36 +35,43 @@
 public interface FurnaceData extends DataManipulator<FurnaceData, ImmutableFurnaceData> {
 
     /**
-     * Gets the {@link MutableBoundedValue} for the remaining burn time of the
-     * {@link Furnace}.
+     * Gets the {@link MutableBoundedValue} for the already passed burn time of the {@link Furnace}.
+     * When the {@link #passedBurnTime()} is equals the {@link #maxBurnTime()} the current used fuel
+     * is depleted.
      *
-     * @return The value for the remaining burn time
+     * @return The value for the already passed burn time
      */
-    MutableBoundedValue<Integer> remainingBurnTime();
+    MutableBoundedValue<Integer> passedBurnTime();
 
     /**
-     * Gets the {@link MutableBoundedValue} for the remaining cook time of the
-     * {@link Furnace}.
+     * Gets the {@link MutableBoundedValue} for the maximum amount of fuel that can be supplied with
+     * the used fuel item.
      *
-     * <p>The {@link #remainingCookTime()} is the difference of {@link #maxCookTime()} and
-     * the time the item has already been cooked.</p>
+     * Its represented by the flame icon in the {@link Furnace}, if the flame is 100% filled the
+     * value is exact this one. So its the maximum of the {@link #passedBurnTime()}.
      *
-     * @return The value for the remaining cook time
+     * @return The value for the maximum amount of fuel that can be supplied with the used fuel item
      */
-    MutableBoundedValue<Integer> remainingCookTime();
+    MutableBoundedValue<Integer> maxBurnTime();
 
     /**
-     * Gets the {@link MutableBoundedValue} for the cook time of the
-     * {@link Item} that should be cooked.
+     * Gets the {@link MutableBoundedValue} for the already passed cook time of the {@link
+     * org.spongepowered.api.item.inventory.ItemStack} in the {@link Furnace}. When the {@link
+     * #passedCookTime()} is equals the {@link #maxCookTime()} the {@link
+     * org.spongepowered.api.item.inventory.ItemStack} is cooked.
      *
-     * <p>This is called "maxCookTime" because the client calculates
-     * {@link #remainingCookTime()} from the "maxCookTime" minus the
-     * time the item cooked already.</p>
+     * @return The value for the already passed cook time
+     */
+    MutableBoundedValue<Integer> passedCookTime();
+
+    /**
+     * Gets the {@link MutableBoundedValue} for the total time the {@link
+     * org.spongepowered.api.item.inventory.ItemStack} have to cook until it is cooked.
      *
-     * <p>This is the maximum of {@link #remainingCookTime()}.</p>
+     * Its represented by the arrow icon in the {@link Furnace}, if the arrow is 100% filled the
+     * value is exact this one. So its the maximum of the {@link #passedCookTime()}.
```

Comment:
This value represents the progress of the arrow icon in the furnace. if the arrow is completely filled, it's value matches this value.